src/plugins/shared-variables/graph/plotted-variables-adornment/plotted-variables.scss

```diff
@@ -0,0 +1,43 @@
+g.plotted-variable {
+  pointer-events: stroke;
+
+  // All 'highlight' elements are invisible unless g.plotted-variable has 'selected' class.
+  .plotted-variable-highlight {
+    opacity: 0;
+  }
+
+  &.selected {
+    .plotted-variable-highlight {
+      opacity: 1;
```

Comment:
Should this be .5?

Reply:
I don't think so; 50% opacity is already set in the stroke colors where needed.